package.json

```diff
@@ -37,12 +35,18 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^6.7.2",
+    "@babel/preset-env": "^7.8.4",
```

Comment:
I think you don't need `@babel/preset-env` and `@babel/preset-react`, they should come with create-react-app

Reply:
Removed